pages/profile/ProfileForm.js

```diff
@@ -0,0 +1,135 @@
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function ProfileForm({
+  initialProfile,
+  avatars,
+  onSave,
+  onCancel,
+}) {
+  const [tempProfile, setTempProfile] = useState({ ...initialProfile });
```

Comment:
you don't need to copy the initialProfile and the name of the state should reflect that it is the form data that is stored here:
```suggestion
  const [formValues, setFormValues] = useState(initialProfile);
```